### PR TITLE
Allow only numerical values for a price field

### PR DESCRIPTION
**Pull request by mic4ael**

### CODE REVIEW

**Comment by ThiefMaster** [comment]
I think we currently allow amounts such as 13.37. Your patch would change this...

**Comment by mic4ael** [comment]
Yeah, I have just noticed that. Fixing that.

File: indico/htdocs/js/indico/RegistrationForm/section.js [hunk 1]
```diff
@@ -705,8 +705,9 @@ ndRegForm.directive("ndSocialEventSection", function() {
                            align: 'center',
                            width: 50,
                            editable: true,
-                           edittype: "text",
+                           edittype: 'text',
                            editoptions: {size: "7", maxlength: "20"},
+                           pattern: '/^(\\d*[.])?\\d+$/'
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/RegistrationForm/section.js]: I think this regex would make more sense:

    ^(\\d+(\\.\\d{2})?)?$

For monetary amounts only 2 digits behind the decimal point make sense.

Comment (mic4ael) [inline indico/htdocs/js/indico/RegistrationForm/section.js]: Yeah, quite reasonable.

Comment (mic4ael) [inline indico/htdocs/js/indico/RegistrationForm/section.js]: Might be better if we don't force to type 2 numbers after the decimal point ^(\\d+(\\.\\d{1,2})?)?$`

Comment (ThiefMaster) [inline indico/htdocs/js/indico/RegistrationForm/section.js]: Does anyone seriously write monetary values with only one digit after the comma?

Comment (mic4ael) [inline indico/htdocs/js/indico/RegistrationForm/section.js]: You can't rule out such possiblity for sure. To me it is just a convenience, not worth a bigger discussion.

**Comment by mic4ael** [comment]
Applied to new registration form.

**Pull request closed without merging**